ISPCPromWeb/myvet
# feat: 12 - Authentication and Roles

### What does this PR do?

With this PR we are adding Authentication for both Client and Stuff User

### What is changing?

- Models
- Views
- Urls
- Backends
- Serializers
- Login
- Dashboard
- routes

### How should it be tested manually?

- Go to `/login` and use a test user to login

**Expected result:** You should be able to see the dashboard page with your email or name on it

### Which are the relevant tickets?

- #12 

### Are there any related PRs?

- N/A

### Screenshots

- N/A
